tests/command_test.py

```diff
@@ -30,3 +33,57 @@ class Rtrace(Command):
 
     rtrace = Rtrace()
     assert rtrace.command == 'rtrace'
+
+def test_path_checker():
+    test_path1=r'test\some directory\some_file.rad'
+    test_path2=r'test\some directory\some_file.bmp'
```

Comment:
In all our tests we use the UNIX format.

```suggestion
    test_path1 = 'test/some directory/some_file.rad'
    test_path2 = 'test/some directory/some_file.bmp'
```